app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def update
+    user = User.find_by(id: params[:id])
+    return render json: { errors: ['User not found'] }, status: :not_found unless user
+
+    if user.update(users_params)
+      render json: { user: user }, status: :ok
+    else
+      render json: { errors: [user.errors.full_messages.join(', ')] }, status: :unprocessable_entity
```

Comment:
Have you looked into Rails' built-in `to_sentence` method? We could potentially replace `user.errors.full_messages.join(', ') `with `user.errors.full_messages.to_sentence`